### PR TITLE
ImportC apply const to correct declarator

### PR DESCRIPTION
**Pull request by WalterBright**
Only handles const. More work is needed for other specifiers.

### CODE REVIEW

**Comment by WalterBright** [comment]
I know what's wrong, will fix it tomorrow.

File: src/dmd/cparse.d [hunk 1]
```diff
@@ -2165,7 +2165,11 @@ final class CParser(AST) : Parser!AST
         out Identifier pident, StorageClass storageClass = 0)
     {
         //printf("cparseDeclarator(%d)\n", declarator);
+        AST.Types constTypes; // all the Types that will need `const` applied to them
+        constTypes.setDim(0);
 
+        AST.Type parseDecl(AST.Type t)
+        {
         AST.Type ts;
```
Comment (ibuclaw) [inline src/dmd/cparse.d:2173]: There's an indentation missing here.

Comment (WalterBright) [reply src/dmd/cparse.d]: I thought I'd minimize the diffs for review, and do the indent later. Ok, I added it now.

Comment (ibuclaw) [reply src/dmd/cparse.d]: I can hide whitespace differences in the UI, so that's not a problem.

**Review by ibuclaw — APPROVED** [review]

**Pull request merged**